product_price_history/readme/USAGE.rst

```diff
@@ -0,0 +1 @@
+Go to ``Inventory > Master data > Product price history`` and handle prices
```

Comment:
In a UX PoV I'll expect this menu in the sale menu and not in a stock menu. Don't you think ?

Reply:
better stock, it refers to costs

Reply:
Oh ! I didnt saw that. In that case, I think name and menu are error prone. Should be product_standard_price_history and standard price history. (Or "cost", that is also used)
Price, in Odoo refers more to sale price.

Reply:
I kept the module name, because it is related to model `product.price.history`, and modified manifest name and summary

Reply:
Thanks !
I understand your PoV regarding the name of the module.

Could you also change the name of the menu ? ``Product Price History`` > ``Product Cost History``.
(the translation is OK (``Storia costo medio prodotto``, but the original name is not)

Reply:
@legalsylvain done, thanks

Reply:
thanks !